src/NServiceBus.Persistence.CosmosDB/Outbox/LogicalOutboxBehavior.cs

```diff
@@ -107,4 +107,4 @@ static AddressTag DeserializeRoutingStrategy(Dictionary<string, string> options)
         readonly JsonSerializer serializer;
         readonly ContainerHolderResolver containerHolderResolver;
     }
-}
\ No newline at end of file
+}
```

Comment:
Do you know where this line ending change is coming from?

Reply:
Likely because it was (re)edited using Github which always adds a CRLF at the end

Reply:
Fixed in 47658d8